css-modules/side-container.less

```diff
@@ -0,0 +1,79 @@
+// General styling of tabs. If it ever gets extracted, these styles should go to common component.
+@outlineColor: #797979;
+@outlineWithoutBottom: 0 -1px 0 @outlineColor, -1px 0 0 @outlineColor, 1px 0 0 @outlineColor;
+
+.tabs {
+  height: 100%;
+  margin-left: 1px;
+}
+
+.tabList {
+  padding-top: 6px;
+  border-bottom: none;
+  margin: 0;
+}
+
+.tab {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  list-style: none;
+  margin-bottom: -1px;
+
+  border-style: solid;
+  border-top-left-radius: 9px;
+  border-top-right-radius: 9px;
+  border-width: 4px;
+  border-bottom: none;
+  background: white;
+  margin-right: 6px;
+  //box-shadow: 0 0 0 1px #797979;
+  vertical-align: bottom;
+  color: #434343;
+  padding: 0;
+  font-size: 14px;
```

Comment:
Font-family for tab labels should be Lato and font-weight should be bold.

Reply:
Thanks, I've updated that. Actually, I've just globally changed the default font from Museo to Lato. I think Museo was my old choice (based on Material UI?). Michael seems to use Lato for all the components.